mypy-nofollow.ini

```diff
@@ -21,7 +21,7 @@ files =
 
 # Minimum version supported - variable annotations were introduced
 # in Python 3.8
-python_version = 3.8
+python_version = 3.11
```

Comment:
Really confused about this update... Should still be 3.8? Is this a bug?

Reply:
uhm, did you see my explanatory comment above? :)